zebra-chain/src/block/header.rs

```diff
@@ -49,7 +49,7 @@ pub struct BlockHeader {
     /// hash must be less than or equal to, in the same nBits format
     /// used by Bitcoin.
     ///
-    /// For a block at block height height, bits MUST be equal to
+    /// For a block at block height, bits MUST be equal to
     /// ThresholdBits(height).
```

Comment:
```suggestion
    /// For a block at block height `height`, bits MUST be equal to
    /// `ThresholdBits(height)`.
```
I think this one was a missing `code` block, the second `height` was supposed to be a reference to the contents of the enum variant.